blueprints/data-solutions/mongodb-cluster/variables.tf

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "cluster_id" {
+  description = "Cluster ID when deploying multiple clusters. Defaults to 'default'."
+  type        = string
+  default     = "default"
+}
+
+variable "data_disk_size" {
+  description = "Size of MongoDB data disks in GB."
+  type        = number
+  default     = 10
+}
+
+variable "healthcheck_image" {
+  description = "Container image for healthcheck."
+  type        = string
+}
+
+variable "network" {
```

Comment:
can you rename to `vpc_config` so that the interface is similar to other recent blueprints?

Reply:
Renamed.